components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/config/APIMConfigServiceImplTestCase.java

```diff
@@ -0,0 +1,88 @@
+package org.wso2.carbon.apimgt.impl.config;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.wso2.carbon.apimgt.api.APIManagementException;
+import org.wso2.carbon.apimgt.impl.dao.SystemConfigurationsDAO;
+import org.wso2.carbon.apimgt.impl.dto.UserRegistrationConfigDTO;
+import org.wso2.carbon.apimgt.impl.utils.APIUtilTest;
+import java.io.File;
+import java.io.IOException;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({APIMConfigServiceImpl.class, SystemConfigurationsDAO.class})
+public class APIMConfigServiceImplTestCase {
+
+    @Test
+    public void testGetSelfSighupConfigWhenSelfSignUpIsEnabled() throws APIManagementException, IOException {
```

Comment:
Typo in the method name. Check other method names as well

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/11919